c2rust-transpile/Cargo.toml

```diff
@@ -26,6 +26,7 @@ indexmap = { version = "1.0.1", features = ["serde-1"] }
 itertools = "0.10"
 libc = "0.2"
 log = "0.4"
+log-reroute = "0.1"
```

Comment:
any reason not to have this at the latest version 0.1.8?

Reply:
I meant to not specify the patch version so it could upgrade without manually bumping it, like the `itertools`, `libc`, and `log` versions above.  Or should I specify the specific the version here?  It's 0.1.8 in `Cargo.lock`.

Reply:
no i think thats a good reason to keep it as-is